governance/votes/Executive vote - January 14, 2022.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+summary: Recover DAI from Optimism escrow, Uniswap LP Vault Debt Floor parameter changes, Recognized Delegate compensation distribution for December.
+date: 2022-01-14T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- 10,000,000 DAI will be transferred from the Optimisim Escrow to a user who accidentally burned DAI on the Optimism chain.
```

Comment:
I tend to write anything over 999,999 as x million DAI. I think it's easier to parse than the set of zeros.

Reply:
Also maybe be consistent on linking to polls / forum posts in the summary. You did for the comp trial, but not for this or the dust thing?

Reply:
Fixed both